Added BODY_SIZE_LIMIT to the env variables recognized by the node adapter (#9326)
sveltejs/kit: .changeset/perfect-brooms-hope.md

---
'@sveltejs/adapter-node': patch
---

> fix:Added BODY_SIZE_LIMIT to the env variables recognized by the node adapter

Reviewer: don't need to put "node adapter" in the text here as it will be in the node-adapter changelog so already implied. also, we typically put a space after the `:` and then start with a lowercase letter
```suggestion
fix: added `BODY_SIZE_LIMIT` to the list of recognized env variables
```

Reviewer 2: should be 'add' not 'added', for consistency with our other changesets

```suggestion
fix: add `BODY_SIZE_LIMIT` to the list of recognized env variables
```
